### PR TITLE
Added GetCurrencyBySymbol

### PR DESCRIPTION
**Pull request by bakingam1983**
Added GetCurrencyBySymbol

### CODE REVIEW

File: CoinmarketcapClient/CoinmarketcapClient.cs [hunk 1]
```diff
@@ -28,11 +28,21 @@ Currency ICoinmarketcapClient.GetCurrencyBySlug(string slug)
             return CurrencyBySlugList(new List<string> {slug}, string.Empty).First();
         }
 
+        Currency ICoinmarketcapClient.GetCurrencyBySymbol(string Symbol)
```
Comment (DominikLindemann) [inline CoinmarketcapClient/CoinmarketcapClient.cs]: please write **s**ymbol in lowercase

File: CoinmarketcapClient/CoinmarketcapClient.cs [hunk 2]
```diff
@@ -28,11 +28,21 @@ Currency ICoinmarketcapClient.GetCurrencyBySlug(string slug)
...
+        {
+            return CurrencyBySymbolList(new List<string> { Symbol }, string.Empty).First();
+        }
+
         Currency ICoinmarketcapClient.GetCurrencyBySlug(string slug, string convertCurrency)
         {
             return CurrencyBySlugList(new List<string> { slug }, convertCurrency).First();
         }
 
+        Currency ICoinmarketcapClient.GetCurrencyBySymbol(string Symbol, string convertCurrency)
```
Comment (DominikLindemann) [inline CoinmarketcapClient/CoinmarketcapClient.cs]: please write **s**ymbol in lowercase

File: CoinmarketcapClient/CoinmarketcapClient.cs [hunk 3]
```diff
@@ -43,6 +53,16 @@ public IEnumerable<Currency> GetCurrencyBySlugList(string[] slugList, string con
             return CurrencyBySlugList(slugList.ToList(), convertCurrency);
         }
 
+        public IEnumerable<Currency> GetCurrencyBySymbolList(string[] SymbolList)
```
Comment (DominikLindemann) [inline CoinmarketcapClient/CoinmarketcapClient.cs]: please write **s**ymbolList in lowercase

File: CoinmarketcapClient/CoinmarketcapClient.cs [hunk 4]
```diff
@@ -43,6 +53,16 @@ public IEnumerable<Currency> GetCurrencyBySlugList(string[] slugList, string con
...
+        {
+            return CurrencyBySymbolList(SymbolList.ToList(), string.Empty);
+        }
+
+        public IEnumerable<Currency> GetCurrencyBySymbolList(string[] SymbolList, string convertCurrency)
```
Comment (DominikLindemann) [inline CoinmarketcapClient/CoinmarketcapClient.cs]: please write **s**ymbolList in lowercase

File: CoinmarketcapClient/CoinmarketcapClient.cs [hunk 5]
```diff
@@ -56,6 +76,19 @@ private IEnumerable<Currency> CurrencyBySlugList(List<string> slugList, string c
             return result;
         }
 
+        private IEnumerable<Currency> CurrencyBySymbolList(List<string> SymbolList, string convertCurrency)
```
Comment (DominikLindemann) [inline CoinmarketcapClient/CoinmarketcapClient.cs]: please write **s**ymbolList in lowercase

File: CoinmarketcapClient/ICoinmarketcapClient.cs [hunk 6]
```diff
@@ -7,11 +7,16 @@ public interface ICoinmarketcapClient
...
 
         Currency GetCurrencyBySlug(string slug);
+        Currency GetCurrencyBySymbol(string symbol);
         Currency GetCurrencyBySlug(string slug, string convertCurrency);
+        Currency GetCurrencyBySymbol(string symbol, string convertCurrency);
 
         IEnumerable<Currency> GetCurrencyBySlugList(string[] slugList);
         IEnumerable<Currency> GetCurrencyBySlugList(string[] slugList, string convertCurrency);
 
+        IEnumerable<Currency> GetCurrencyBySymbolList(string[] SymbolList);
```
Comment (DominikLindemann) [inline CoinmarketcapClient/ICoinmarketcapClient.cs]: please write **s**ymbolList in lowercase

File: CoinmarketcapClient/ICoinmarketcapClient.cs [hunk 7]
```diff
@@ -7,11 +7,16 @@ public interface ICoinmarketcapClient
...
+        IEnumerable<Currency> GetCurrencyBySymbolList(string[] SymbolList, string convertCurrency);
```
Comment (DominikLindemann) [inline CoinmarketcapClient/ICoinmarketcapClient.cs]: please write **s**ymbolList in lowercase

**Review by DominikLindemann — CHANGES_REQUESTED** [review]
please use camelCase overall

**Comment by bakingam1983** [comment]
I' ve made in the second commit...

**Review by DominikLindemann — APPROVED** [review]

**Pull request merged**